vendor/build-logger/src/ldlogger-tool-gcc.c

```diff
@@ -227,6 +231,23 @@ static void getDefaultArguments(const char* prog_, LoggerVector* args_)
   pclose(cmdOut);
 }
 
+char* findFullPath(const char* executable, char* fullpath) {
```

Comment:
Since `fullpath` is already an output parameter, maybe it would be better to just return a bool to indicate success.

Reply:
with this, I am followign the realpath calling convention http://man7.org/linux/man-pages/man3/realpath.3.html